integration-tests/src/test/set-collect-coins-contract.test.ts

```diff
@@ -0,0 +1,46 @@
+import { KeyringPair } from 'creditcoin-js';
+import { POINT_01_CTC } from '../constants';
+import { creditcoinApi } from 'creditcoin-js';
+import { CreditcoinApi } from 'creditcoin-js/lib/types';
+import { testData } from './common';
+import { extractFee, testIf } from '../utils';
+
+describe('SetCollectCoinsContract', (): void => {
+    let ccApi: CreditcoinApi;
+    let sudoSigner: KeyringPair;
+
+    const { keyring } = testData;
+
+    beforeAll(async () => {
+        ccApi = await creditcoinApi((global as any).CREDITCOIN_API_URL);
+        if ((global as any).CREDITCOIN_EXECUTE_SETUP_AUTHORITY) {
+            sudoSigner = keyring.addFromUri('//Alice');
+        }
+    });
+
+    afterAll(async () => {
+        await ccApi.api.disconnect();
+    });
+
+    testIf((global as any).CREDITCOIN_EXECUTE_SETUP_AUTHORITY, 'fee is min 0.01 CTC', async (): Promise<void> => {
+        const { api } = ccApi;
+
+        /* eslint-disable @typescript-eslint/naming-convention */
+        const contract = api.createType('PalletCreditcoinGCreContract', {
```

Comment:
@CAGS295 I tried regenerating typedefs but there were no changes detected. How do I build this type on the client side before submitting the extrinsic ?